ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```diff
@@ -124,6 +124,42 @@ objects:
 
         mkdir -p "${HOME}"
 
+        # Install python-openstackclient which is currently required to run
+        # some tests (in parallel mode). See: https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/drivers/in_tree.go#L956
+        cat > /etc/yum.repos.d/rdo-stein.repo <<-"EOF_CAT"
+        [openstack-stein]
+        name=OpenStack Stein Repository
+        baseurl=http://mirror.centos.org/centos/7/cloud/$basearch/openstack-stein/
+        gpgcheck=1
+        enabled=1
+        gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-Cloud
+        EOF_CAT
+
+        cat > /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-Cloud <<-"EOF_CAT"
+        -----BEGIN PGP PUBLIC KEY BLOCK-----
+        Version: GnuPG v2.0.22 (GNU/Linux)
+         
+        mQENBFVWcCcBCACfm3eQ0526/I0/p7HpR0NjK7K307XHhnbcbZv1sDUjQABDaqh0
```

Comment:
For future reference, this is:

```console
$ gpg --with-fingerprint cert-extracted-from-template.pub
pub  2048R/764429E6 2015-05-15 CentOS Cloud SIG (http://wiki.centos.org/SpecialInterestGroup/Cloud) <security@centos.org>
      Key fingerprint = 736A F511 6D9C 40E2 AF6B  074B F9B9 FEE7 7644 29E6
```

from [here][1].

[1]: https://www.centos.org/keys/#cloud-sig